src/pages/task/logs/LogTypes.tsx

```diff
@@ -1,26 +1,41 @@
 import React from "react";
 import {
-  AgentLogsQuery,
-  EventLogsQuery,
   GET_AGENT_LOGS,
   GET_EVENT_LOGS,
   GET_SYSTEM_LOGS,
   GET_TASK_LOGS,
-  LogMessage,
+} from "gql/queries/get-task-logs";
+import {
+  EventLogsQuery,
+  EventLogsQueryVariables,
   SystemLogsQuery,
-  TaskEventLogEntry,
+  SystemLogsQueryVariables,
+  AgentLogsQuery,
+  AgentLogsQueryVariables,
   TaskLogsQuery,
-} from "gql/queries/get-task-logs";
+  TaskLogsQueryVariables,
+  LogMessage,
+  TaskEventLogEntry,
+} from "gql/generated/types";
 import { useQuery } from "@apollo/react-hooks";
 import { TaskEventLogLine } from "./logTypes/TaskEventLogLine";
 import { LogMessageLine } from "./logTypes/LogMessageLine";
 import { ApolloError } from "apollo-client";
 import { useParams } from "react-router-dom";
 import get from "lodash/get";
 
+interface TaskEventLogEntryType extends TaskEventLogEntry {
+  kind?: "taskEventLogEntry";
```

Comment:
This doesn't exist on the official graphql type but it did in our types. enabling the typeName flag would probably be useful here? But im not sure how often we use it. What do you think?

Reply:
@SupaJoon